### PR TITLE
add book detail view and test to signUp

### PR DESCRIPTION
**Pull request by joseleonardorojas**
## Summary

* Add book detail view
* Test to signUp

## Screenshots

image

## Trello Card

https://trello.com/c/lzf2O2r8/35-tests-de-sign-up
https://trello.com/c/03vN05Ly/37-book-detail

### CODE REVIEW

File: src/components/BookList/index.tsx [hunk 1]
```diff
@@ -10,17 +11,20 @@ import styles from './styles.module.scss';
...
       ) : (
         books.map((book: IBook) => (
-          <div key={book.id} className={styles.cardBook}>
-            <img className={styles.imgBook} src={book.image_url} />
-            <span className={styles.titleBook}>{book.title}</span>
-            <span className={styles.authorBook}>{book.author}</span>
-          </div>
+          <Link to={`/books/${book.id}`} key={book.id}>
+            <div className={styles.cardBook}>
+              <img className={styles.imgBook} src={book.image_url} />
```
Comment (DavidRojas1612) [inline src/components/BookList/index.tsx]: dont forget add to tag img the alt attribute

File: src/screens/BookDetail/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          {isLoading ? (
+            <Loading />
+          ) : (
+            <div className={styles.detailBook}>
+              <div className={styles.sectionImg}>
+                <img className={styles.imgBook} src={book?.image_url} />
+                <img className={styles.imgBadge} src={imgBadge} />
+              </div>
+              <div className={styles.infoBook}>
+                <div>
```
Comment (DavidRojas1612) [inline src/screens/BookDetail/index.tsx]: dont use empty div, they are unnecessary

File: src/screens/BookDetail/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                  <span className={styles.titleBook}>{book?.title}</span>
+                  <span className={styles.textGenre}>{`(${book?.genre})`}</span>
+                </div>
+                <div className={styles.bar} />
+                <div>
```
Comment (DavidRojas1612) [inline src/screens/BookDetail/index.tsx]: same

**Review by DavidRojas1612 — CHANGES_REQUESTED** [review]

File: src/screens/SignUp/index.tsx [hunk 4]
```diff
@@ -30,8 +30,8 @@ function SignUp() {
 
   const mutation = useMutation(signUp, {
     onSuccess: data => {
-      console.log(data);
       if (data.ok) {
+        console.log(data.data)
```
Comment (betoobernal) [inline src/screens/SignUp/index.tsx]: remove this log

**Pull request merged**